Src/CsvLINQPadDriver/ConnectionDialog-NoWPF.cs

```diff
@@ -1,21 +1,16 @@
-﻿using System;
```

Comment:
I wasn't really sure about changes in this file. I just made unnecessary fixes to fix the compilation.

Reply:
Just realized this file wasn't compiled before at all, so the changes are wrong I should rather exclude this file from the library compilation, right?